docs/nodes/credentials/Zendesk/README.md

```diff
@@ -15,31 +15,54 @@ Create a [Zendesk](https://zendesk.com/) account.
 
 ## Using OAuth
 
-::: tip ⛅️ Note for n8n.cloud users
-You'll only need to enter the Credentials Name and click on the circle button in the OAuth section to connect your Zendesk account to n8n.
-:::
-
 1. Access your Zendesk dashboard.
```

Comment:
Is there a generic link to the dashboard that we can share?

Reply:
Zendesk doesn't provide a generic link.